JsonBodyListener.php

```diff
@@ -41,6 +48,11 @@ public function onKernelRequest(GetResponseEvent $event)
                 throw new BadRequestHttpException('Invalid ' . $format . ' message received');
             }
 
+            $jsonSchema = $request->get('_jsonSchema');
```

Comment:
We maybe should rename it `_requestJsonSchema` if for example we plan some day to add a `_responsJsonSchema` for any reason?
I'm still not sure it's a good idea.